Sources/Auth/Types.swift

```diff
@@ -71,7 +71,7 @@ public struct Session: Codable, Hashable, Sendable {
 
   /// UNIX timestamp after which the ``Session/accessToken`` should be renewed by using the refresh
   /// token with the `refresh_token` grant type.
-  public var expiresAt: TimeInterval?
+  public var expiresAt: TimeInterval
```

Comment:
@hf @kangmingtay or @J0 

Is this guarantee to always be returned in the Session? It was optional for some reason on swift.

Reply:
Should be - or at least we guarantee it in the spec: https://github.com/supabase/auth/blob/master/openapi.yaml#L1920

I think we added the `ExpiresAt` after the other properties which might be why the author defined it as optional then for backward compatibility during the transition from an older Auth version to the newer one